pyasic/miners/models/epic/blockminer/blockminer.py

```diff
@@ -0,0 +1,7 @@
+from pyasic.miners.makes import ePICMake
+
+
+class Blockminer520(ePICMake):
+    raw_model = "Blockminer 520i"
```

Comment:
Any reason to not name this `Blockminer520i` instead?

Reply:
None other than me missing the i...good catch